content/news/XXXX-XX-XX-mixxx-2-4-0-features.md

```diff
@@ -0,0 +1,85 @@
+title: Mixxx 2.4 Release
+authors: Jörg Wartenberg
+status: draft
+tags: release, 2.4
+comments: yes
+
+
+We are proud to announce Mixxx 2.4, which will be released NEXT WEEK, a new major version of our free open source DJ software. It took about two years since the last release due to the large number of new features and changes under the hood.
```

Comment:
Even more nitpicky, but I would probably avoid the phrasing with "our". Given that users will likely read the blog post far into the future, I would also leave out the "next week" and just release the blog post simultaneously with the actual binaries:

```suggestion
We are proud to announce Mixxx 2.4, the next major release of the free and open source DJ software. This long awaited version includes many new features and improvements under the hood.
```

Reply:
This is the phrasing for the press information, that we will distribute in advance to the release. To give the DJ press the time to prepare their articles, videos etc.
Later we reuse the text for the blogpost and change this phrasing as you suggested.

Reply:
I am ok with this being in the future tense, but I'd prefer not to promise a specific date because we really can't guarantee that.  Also don't use all-caps for emphasis, use bold or italics.
```suggestion
We are proud to announce a new major version of Mixxx which will be released in the next couple weeks. Version 2.4 took about two years since the last release due to the large number of new features and changes under the hood.
```

Reply:
NEXT WEEK is just a place holder

Reply:
ok cool. for placeholders, the traditional keyword is "TK" or even "TKTKTKTKTKTK" (https://stevenpressfield.com/2016/05/the-magic-of-tk/)